.stylelintrc

```diff
@@ -0,0 +1,21 @@
+{
+    "plugins": [
+        "stylelint-scss"
+    ],
+    "extends": [
+        "stylelint-config-standard"
+    ],
+    "rules": {
+        "at-rule-no-unknown": null,
+        "no-descending-specificity": null,
+        "property-no-unknown": [
+            true,
+            {"ignoreSelectors": [":export"]}
+        ],
+        "scss/at-rule-no-unknown": true,
+        "selector-pseudo-class-no-unknown": [
+            true,
+            {"ignorePseudoClasses": ["export"]}
```

Comment:
The two rule customizations related to `export` cover an edge-case usage we have in https://github.com/xh/hoist-react/blob/develop/desktop/cmp/grid/columns/Actions.scss where we do some fancy stuff to make a value importable from JS.